### PR TITLE
[PS-2841] remove SweetAlert2 dependency

### PR DESCRIPTION
**Pull request by willmartian**
## Type of change

```
- [ ] Bug fix
- [ ] New feature development
- [x] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Remove unused SweetAlert2 dependency; relies on https://github.com/bitwarden/clients/pull/6170 and https://github.com/bitwarden/clients/pull/6138.

## Code changes

- **file.ext:** Description of what was changed and why

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

**Review by Hinton — COMMENTED** [review]
There are some usages in the following files which we can remove as well.

- `apps/web/src/scss/base.scss`
- `apps/web/src/scss/buttons.scss`

**Review by Hinton — APPROVED** [review]

**Comment by willmartian** [comment]
No code change, just rebased to master.

File: apps/web/src/app/app.component.ts [hunk 1]
```diff
@@ -198,19 +196,6 @@ export class AppComponent implements OnDestroy, OnInit {
       });
     });
 
-    this.router.events.pipe(takeUntil(this.destroy$)).subscribe((event) => {
-      if (event instanceof NavigationEnd) {
-        const modals = Array.from(document.querySelectorAll(".modal"));
-        for (const modal of modals) {
-          (jq(modal) as any).modal("hide");
-        }
```
Comment (Hinton) [inline apps/web/src/app/app.component.ts:206]: I think we still need this for bootstrap modals.

Comment (willmartian) [reply apps/web/src/app/app.component.ts]: Fixed in [2a24522](https://github.com/bitwarden/clients/pull/6191/commits/2a24522879b45937ba83935a4c9381dcd7b24369)

**Review by Hinton — APPROVED** [review]

**Pull request merged**